Dockerfile

```diff
@@ -1,17 +1,41 @@
-FROM alpine:latest
+FROM golang:1.12.5-alpine AS build
```

Comment:
could you change this to `1.12.4` for the time being please, until I update everything across the board to use `1.12.5`.

Reply:
Done, I followed the version on the GolangCI line in the [Makefile](https://github.com/markphelps/flipt/blob/cc158f9/Makefile#L14) but missed the other variables, my bad.